qutebrowser/browser/webengine/webview.py

```diff
@@ -258,13 +259,51 @@ def acceptNavigationRequest(self,
         self.navigation_request.emit(navigation)
         return navigation.accepted
 
+    @staticmethod
+    def extra_suffixes_workaround(upstream_mimetypes):
+        """Return any extra suffixes for mimetypes in upstream_mimetypes.
+
+        Return any file extensions (aka suffixes) for mimetypes listed in
+        upstream_mimetypes that are not already contained in there.
+
+        # WORKAROUND: for https://bugreports.qt.io/browse/QTBUG-116905
+        # Affected Qt versions > 6.2.2 (probably) < 6.7.0
```

Comment:
```suggestion
        WORKAROUND for https://bugreports.qt.io/browse/QTBUG-116905
        Affected Qt versions > 6.2.2 (probably) < 6.7.0
```

doesn't need a comment sign I guess?